jep/README.md

```diff
@@ -86,63 +86,63 @@ This jupyter-ui-toolkit repository contains three packages:
+- [UI Toolkit](https://github.com/jupyterlab-contrib/jupyter-ui-toolkit) is a component library for building web interfaces in Jupyter ecosystem (JupyterHub, Jupyter Widgets, JupyterLab, etc.) This JEP proposes to build on top of this implementation.
 
 # Unresolved questions
 
 Base components are in-scope of this JEP. Reactivity and collaboration/local-first are out-of-scope of this JEP.
 
-It should be possible for a React.js widget to share a global state with a widget developed in another technology (Vue.js, Svelte...). This can be achieved by a adequate global state/store system that is not restricted a single technology (so should be `polyglot`). This can be defined in subsequent JEP.
+It should be possible for a React.js widget to share a global state with a widget developed in another technology (Vue.js, Svelte...). This can be achieved by an adequate global state/store system that is not restricted to a single technology (so should be **polyglot**). This can be defined in a subsequent JEP.
 
-Realtime collaboration and local-first applications are not in-scope for now but it is logical to mention that and to anticipate they will depend on a CRDT solution (e.g. Y.js in use for now in JupyterLab). We could imagine a global state being responsible to synchronize the model via CRDT messages. Such implementations are already being worked on with well-know global state systems like Redux, with e.g. [Y.js binding to Redux](https://github.com/lscheibel/redux-yjs-bindings). The UI components should thus be able to easily bind to such market standard global states.
+Realtime collaboration and local-first applications are not in-scope for now but it is logical to mention that and to anticipate that they will depend on a CRDT solution (Conflict-free replicated data types). For example, Y.js is in use for now in JupyterLab. We could imagine a global state to synchronize the model via CRDT messages. Such implementations are already being worked on with well-know global state systems like Redux, with e.g. [Y.js binding to Redux](https://github.com/lscheibel/redux-yjs-bindings). The UI components should thus be able to easily bind to such industry standard global state managers.
 
-Reactivity to support features like [Jupyter Widgets](https://github.com/jupyter-widgets) can be defined in subsequent JEP. The current implementation of Jupyter Widgets relies on [Backbone.js](https://backbonejs.org) for which a successor would be welcome. We could imagine relying on the collaborative feature based on CRDT (Conflict-free replicated data types).
+Reactivity to support features like [Jupyter Widgets](https://github.com/jupyter-widgets) can be defined in a subsequent JEP. The current implementation of Jupyter Widgets relies on [Backbone.js](https://backbonejs.org) for which a successor would be welcome. We could imagine the collaborative feature based on CRDT.
```

Comment:
Once again, the connection between reactivity and this proposal escapes me.

Reply:
I will work on that after merging your PR.